taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -91,21 +91,33 @@ def build_browsertime_task(config, tasks):
         task["run"]["command"].append("--test={}".format(test_name))
         task["run"]["command"].extend(task.pop("args", []))
 
+        # Setup treherder symbol
+        symbol = test_name
+
+        # taskcluster is merging task attributes with the default ones
+        # resulting the --cold extra option in the ytp warm tasks
+        if "youtube-playback" in task["name"]:
+            task["run"]["command"].remove("--cold")
+            symbol = test_name.replace("youtube-playback-", "ytp-")
+
+        # Setup chimera for combined warm+cold testing
+        if task.pop("chimera", False):
+            task["run"]["command"].append("--chimera")
+
+        # Add '-c' to taskcluster symbol when running cold tests
+        if "--cold" in task["run"]["command"]:
```

Comment:
We should get rid of this since we only have chimera tests running here.

Reply:
yes but by default we run cold and we should add -c if we have "--cold" in the command line

Reply:
Ok, then you should also check to make sure chimera isn't enabled here before appending the -c

Reply:
added elif